.gitignore

```diff
@@ -9,8 +9,10 @@
 .coverage
 .coverage.*
 .eggs/
+.idea/
 .installed.cfg
```

Comment:
This file is generated, should not be touched in a pull request

Reply:
Fixed